dime-coding-standards/checklists/Stata Code Review Checklist.md

```diff
@@ -0,0 +1,108 @@
+
+# Stata Peer Code Review Checklist
+
+Use this checklist to improve your own Stata coding practice or to guide your peer code reviews.
+
+
+## Main Do-file Setup
+    
+- [ ] Includes `clear all` and `set more off` at the start.
+- [ ] Sets core configurations (`version`, `matsize`, `varabbrev`) directly or via a wrapper (e.g., `ieboilstart`)
+- [ ] Script runs from start to end after changing directory paths in one place only.
+- [ ] Uses only **relative paths** (no `C:/...`) 
+- [ ] Uses forward slashes in file path for OS compatibility
+- [ ] Installs required packages or includes an `ado` folder with dependencies.
+- [ ] Sets a **random seed** for reproducibility.
+- [ ] Uses `run` or `do` to call modular scripts for cleaning, construction, etc.
+
+
+
+    
+## Data Management
+    
+- [ ] Dataset includes a **unique ID** and is sorted.
```

Comment:
```suggestion
- [ ] Each dataset includes a **unique ID** and is sorted. If the same unit of observation is used in multiple datasets, then the same unique ID should be used for the same observation across those datasets.
```

This might be too long for this brief checklist. Maybe split into to check items. But this point is important to make.